app/src/main/java/org/sopt/and/component/BackTopBar.kt

```diff
@@ -0,0 +1,63 @@
+package org.sopt.and.component
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.KeyboardArrowLeft
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.and.R
+import org.sopt.and.ui.theme.FirstGrey
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BackTopBar(
+    @DrawableRes titleImg:  Int,
+    onBtnClick: () -> Unit,
+    containerColor: Color = FirstGrey,
+    titleContentColor: Color = Color.White,
+    navigationIconContentColor: Color = Color.White
```

Comment:
```suggestion
fun BackTopBar(
    @DrawableRes titleImg:  Int,
    onBtnClick: () -> Unit,
    modifier: Modifier = Modifier
    containerColor: Color = FirstGrey,
    titleContentColor: Color = Color.White,
    navigationIconContentColor: Color = Color.White
```
컴포넌트라는게 여러 컴포저블에서 호출될 수 있는거니까 기본적으로 Modifier를 인자로 받는게 좋습니다!

저는 보통 최상위 컴포저블에 인자로 받은 Modifier를 할당해서 사용하는 편입니다!

또한 Modifier는 기본값이 있는 선택 인자로 받는 편이고, 함수 인자의 순서는 "필수 인자 -> Modifier -> 선택 인자" 순으로 적으시면 됩니다!

Reply:
그렇게 생각을 못했네요,,,! 컴포저블이 여러 컴포저블에서 호출하는거니까 인자로 받는게 좋겟네요 ㅎㅎ
함수 인자 순서도 생각하면서 해보겠습니다!